ihatemoney/models.py

```diff
@@ -434,6 +434,7 @@ def delete(self, project, id):
     date = db.Column(db.Date, default=datetime.now)
     creation_date = db.Column(db.Date, default=datetime.now)
     what = db.Column(db.UnicodeText)
+    tag = db.Column(db.UnicodeText)
```

Comment:
Migration is missing for this new column

Reply:
So we actually had a ton of issues with the migration. We tried running `make create-database-revision` like it says in the Contributing Guidelines on several clones of the repo with our changes, but the script would not work. It would write code to create a table with all the columns including tags. We would then fail tests because "project table already exists" and we'd be unable to run the server because somehow oppositely it wouldn't exist. Then we tried to write/edit the migration file to only add/drop the new tag column but then we'd fail the tests because "duplicate column name: tag" and the server/site wouldn't work either.  Not doing a migration was the only way we were able to get into the application and see that our changes were working... Not sure what the issue was; any ideas?